src/index.html

```diff
@@ -11,8 +11,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="../src/images/imac.jpeg"
+        alt="img"
+        class="card__image"
+      />
+      <div class="card__apple">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__review-container">
+        <div class="stars stars--4">
```

Comment:
Make sure that the `stars` block is implemented according to BEM methodology, where `stars--4` should indicate that there are 4 active stars. Each star should also be an `:after` element of a `stars__star` block as per the task requirements.